### DISCUSSION TITLE
When opiate addiction, Naloxone is not suggested in health window

### DISCUSSION THREAD

**Discussion by kaedepon** [FakeFishGames/Barotrauma#14318] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Naloxone is not suggested for opiate addiction, so medical bots can not use it.
But game files and wiki said that Naloxone can treatment in opiate addiction.

![opiateaddiction](https://github.com/user-attachments/assets/97da422c-d9f0-4688-8e86-16b38fc6aa6a)

### Reproduction steps

1. Command `giveaffliction "Opiate addiction" 100`
2. Watch health window

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
lol, i didn't know it's supposed to heal that :BaroDev:
It's not mentioned in the description and it's not suggested as treatment
![Screenshot_3](https://github.com/user-attachments/assets/0ed8501b-97f5-45d4-b1e6-c2aeeb14cfdf)
But game files and wiki said that it should 
![barodev small](https://github.com/user-attachments/assets/bf30b01a-3a22-4d21-bcd9-48d5cf74224d)

**Reply by kaedepon** [reply]
Thank you for your information! I hadn't checked naloxone's description.
I don't know what is it truth, bad I believe barowiki :-)

**Comment by Regalis11** [comment]
This has now been merged to our private repo. Didn't make it in time for the next update I'm afraid, but will be included in the one after that.
